openreview/venue/invitation.py

```diff
@@ -166,6 +172,9 @@ def set_submission_invitation(self):
             duedate = submission_duedate,
             expdate = tools.datetime_millis(submission_stage.exp_date) if submission_stage.exp_date else None,
             content = {
+                'users_to_notify': {
+                    'value': groups_to_notify
+                },
```

Comment:
I don't think we can remove `email_authors`, `email_pcs`, etc since the venue is using this. I am not sure what's the best way to not make this confusing for PCs:
1. Keep both for all invitations (PCs would be able to edit either field to edit notification settings)
2. Keep  booleans only for normal venues and use `users_to_notify` only for the new config invitations